zscaler/assets/dashboards/zscaler_overview.json

```diff
@@ -0,0 +1 @@
tion":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"@zscaler.Recordtype","sort":{"aggregation":"count","order":"desc"},"limit":10}]}]}]},"layout":{"x":0,"y":0,"width":4,"height":2}},{"id":6845510121838372,"definition":{"title":"Tunnel Types","title_size":"16","title_align":"left","type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":10,"order":"desc"}}],"response_format":"scalar","queries":[{"search":{"query":"source:zscaler @sourcetype:zscalernss-tunnel $ClientIP $DestinationIP $UserID $HTTPHost $HTTPProtocol $HTTPContentType $ZscalerAppClass $Outcome $DNSRequestName"},"data_source":"logs","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"@zscaler.tunneltype","sort":{"aggregation":"count","order":"desc"},"limit":10}]}]}]},"layout":{"x":4,"y":0,"width":4,"height":2}},{"id":1331890568880528,"definition":{"type":"note","content":"Tunnel logs: View data about GRE and IPSec tunnels, such as their health, status, and authentication and encryption algorithms. For more information on the fields available, check out [Zscaler documentation](https://help.zscaler.com/zia/tunnel-insights-logs-columns)\n","background_color":"vivid_yellow","font_size":"16","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true},"layout":{"x":8,"y":0,"width":4,"height":2}},{"id":7464658587878514,"definition":{"title":"Zscaler Tunnel Logs","title_size":"16","title_align":"left","type":"log_stream","indexes":[],"query":"source:zscaler @sourcetype:zscalernss-tunnel $ClientIP $DestinationIP $UserID $HTTPHost $HTTPProtocol $HTTPContentType $ZscalerAppClass $Outcome $DNSRequestName","sort":{"column":"time","order":"desc"},"columns":["@usr.id","@zscaler.Recordtype","@zscaler.tunneltype","@network.destination.ip","@network.client.ip","@network.client.port","@zscaler.rxbytes"],"show_date_column":true,"show_message_column":false,"message_display":"inline"},"layout":{"x":0,"y":2,"width":12,"height":3}}]},"layout":{"x":0,"y":18,"width":12,"height":6}}],"template_variables":[{"name":"ClientIP","default":"*","prefix":"@network.client.ip","available_values":[]},{"name":"DestinationIP","default":"*","prefix":"@network.destination.ip","available_values":[]},{"name":"UserID","default":"*","prefix":"@usr.id","available_values":[]},{"name":"HTTPHost","default":"*","prefix":"@http.url_details.host","available_values":[]},{"name":"HTTPProtocol","default":"*","prefix":"@zscaler.protocol","available_values":[]},{"name":"HTTPContentType","default":"*","prefix":"@zscaler.contenttype","available_values":[]},{"name":"ZscalerAppClass","default":"*","prefix":"@zscaler.appclass","available_values":[]},{"name":"Outcome","default":"*","prefix":"@evt.outcome","available_values":[]},{"name":"DNSRequestName","default":"*","prefix":"@zscaler.dns_req","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"tee-pdp-wqf"}
```

Comment:
Looks like the description hasn't been added.

Reply:
Ah, thank you. I copy-pasted the dashboard JSON from Zscaler's org, so this must've gotten lost in translation. Updated the title in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e